### PR TITLE
Test for v63003/gtm8617 (Tests GTM-8617 in V63003)

### PR DESCRIPTION
**Pull request by VinayYottaDB**
Tests the following release note:

The MUPIP SET command supports the following qualifiers: -N[ULL_SUBSCRIPTS]={never, always, existing}, which controls whether GT.M accepts null subscripts for database keys. -[NO]STD[NULLCOLL], which determines whether GT.M will use standard MUMPS collation or GT.M collation for null-subscripted keys. Previously, this functionality was only available through GDE for database file creation, and DSE for existing database files. FIS strongly recommends avoiding the use of DSE when there is an alternative. (GTM-8617)

### CODE REVIEW

File: v63003/instream.csh [hunk 1]
```diff
@@ -19,13 +19,14 @@
 # gtm8186	    [vinay] Test DO, GOTO and ZGOTO can take offset without a label
 # gtm8804	    [vinay] Test zshow "t" produces only a summary of zshow "g" and "l"
 # gtm8832	    [vinay] Test string literal evaluating to >=1E47 produces a NUMOFLOW error
+# gtm8617	    [vinay] Tests functionality of STDNULLCOLL and NULL_SUBSCRIPTS
```
Comment (nars1) [inline v63003/instream.csh]: STDNULLCOLL -> MUPIP SET command for STDNULLCOLL

File: v63003/u_inref/gtm8617.csh [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+echo "# Null collation set to GT.M"
+$gtm_tst/com/dbcreate.csh mumps 1 >>& db.txt
+$ydb_dist/mupip set -region DEFAULT -std
+$gtm_tst/com/dbcheck.csh >& db.txt
+
+echo "# Null collation set to M"
+$gtm_tst/com/dbcreate.csh mumps 1 >>& db.txt
+$ydb_dist/mupip set -region DEFAULT -nostd
+$gtm_tst/com/dbcheck.csh >& db.txt
+
```
Comment (nars1) [inline v63003/u_inref/gtm8617.csh]: Please remove the huge block of empty lines at the tail of this file.

File: v63003/u_inref/gtm8617.csh [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+# Case for each functionality described in the release note
+#
+
+echo "# Null subscript set to never"
+$gtm_tst/com/dbcreate.csh mumps 1 >>& db.txt
+$ydb_dist/mupip set -region DEFAULT -N=never
```
Comment (nars1) [inline v63003/u_inref/gtm8617.csh]: Please expand qualifier names in test scripts even though abbreviated forms are accepted. i.e. -N -> -NULL_SUBSCRIPTS, -std -> -stdnullcoll etc. in various places.

Comment (nars1) [reply v63003/u_inref/gtm8617.csh]: The output of the MUPIP SET command is included in the reference file. That is not enough proof that null subscripts is set to never. One needs to do a **dse dump -file |& grep "Null subscripts"** to verify that is indeed set in the db file header. Same with the other commands/qualifiers in this test.

File: v63003/u_inref/gtm8617.csh [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+echo "# Null subscript set to existing"
+$gtm_tst/com/dbcreate.csh mumps 1 >>& db.txt
+$ydb_dist/mupip set -region DEFAULT -N=existing
+$gtm_tst/com/dbcheck.csh >& db.txt
+
+echo "# Null subscript set to always"
+$gtm_tst/com/dbcreate.csh mumps 1 >>& db.txt
+$ydb_dist/mupip set -region DEFAULT -N=always
+$gtm_tst/com/dbcheck.csh >& db.txt
+
```
Comment (nars1) [inline v63003/u_inref/gtm8617.csh]: Also add a test that NULLSUBS=<string> where string is none of the allowed 3 choices issues an error.

File: v63003/u_inref/gtm8617.csh [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+echo "# Null collation set to GT.M"
+$gtm_tst/com/dbcreate.csh mumps 1 >>& db.txt
+$ydb_dist/mupip set -region DEFAULT -std
```
Comment (nars1) [inline v63003/u_inref/gtm8617.csh]: One needs to do a **dse dump -file |& grep "Standard Null Collation"** to verify that is indeed set in the db file header. Same with the other commands/qualifiers in this test.
